Hadrons/Modules/MRHQ/RHQInsertionI.hpp

```diff
@@ -35,26 +36,26 @@
 BEGIN_HADRONS_NAMESPACE
 
 /******************************************************************************
- *                                 RHQInsertionI                                      *
+ *                              RHQInsertionI                                 *
  ******************************************************************************/
 BEGIN_MODULE_NAMESPACE(MRHQ)
 
 class RHQInsertionIPar: Serializable
 {
 public:
     GRID_SERIALIZABLE_CLASS_MEMBERS(RHQInsertionIPar,
-                                    std::string, q,
-                                    std::string, gauge,
-                                    unsigned int, gauge_index);
+                                    std::string,    q,
+                                    unsigned int,   index,
+                                    Gamma::Algebra, gamma5,
+                                    std::string,    gauge);
 };
 
 template <typename FImpl, typename GImpl>
 class TRHQInsertionI: public Module<RHQInsertionIPar>
 {
 public:
     BASIC_TYPE_ALIASES(FImpl,);
-    GAUGE_TYPE_ALIASES(GImpl,)
-    SINK_TYPE_ALIASES();
+    GAUGE_TYPE_ALIASES(GImpl,);
 public:
     // constructor
     TRHQInsertionI(const std::string name);
```

Comment:
I would add the line
`MODULE_REGISTER_TMP(RHQInsertionII, ARG(TRHQInsertionI<FIMPL, GIMPL>), MRHQ);`
After the Module register a little bit after this, and not use the RHQInsertionII hpp and cpp files, unless there is an issue with this?